Community/alumni.md

```diff
@@ -0,0 +1,357 @@
+
+
+# CHAOSS Mentorship Alumni
+
+The CHAOSS Community thanks all mentees and mentors who participated in Google Summer of Code and Outreachy.
+
+
+## CHAOSS GSoC Students in 2020
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Keanu Nichols](#keanu-nichols)|Reporting of CHAOSS Metrics|Sean Goggins, Jesus Gonzalez-Barahona|
+|[Pranjal Aswani](#pranjal-aswani)|Reporting of CHAOSS Metrics: Refactoring the existing code and extending the capabilities of the Manuscripts Project|Valerio Cosentino, Jesus Gonzalez-Barahona|
+|[Parth Sharma](#parth-sharma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins|
+|[Bingwen Ma](#bingwen-ma)|Build CHAOSS Risk and Growth Maturity and Decline Metrics in Augur|Sean Goggins, Matt Germonprez|
+|[Aniruddha Karajgi](#aniruddha-karajgi)|Implementing CHAOSS Metrics with Perceval|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Nishchith K Shetty](#nishchith-k-shetty)|Support of Source Code Related Metrics|Jesus Gonzalez-Barahona, valcos, Pranjal Aswani|
+|[Ria Gupta](#Ria-Gupta)|Social Currency Metric System|Valerio Cosentino, Samantha Venia, Logan|
+|[Tianyi Zhou](#Tianyi-Zhou)|Large Social Network Analysis and Anomaly Detection with Augur|Sean Goggins, Jonahz, Gabe Heim|
+|[Sarit Adhikari](#Sarit-Adhikari)|Machine Learning for Anomaly Detection in Open Source Communities|Sean Goggins, Carter Landis, Gabe Heim|
+|[Abhinav Bajpai](#Abhinav-Bajpai)|Implementing GitLab Data Collection Worker and Mapper to bind the responses of GitLab API, Github API & the Augur schema|Sean Goggins, Carter Landis|
+|[bistaastha](#bistaastha)|Build Workflow process for CHAOSS D&I Badging Project|Matt Snell|
+|[vchrombie](#vchrombie)|Creating Quality models using GrimoireLab and CHAOSS metrics|Valerio Cosentino, Aniruddha Karajgi|
+|[Saicharan Reddy](#Saicharan-Reddy)|Implementation of GitLab Data Collection Worker & Test Coverage Improvement|Elita Nelson, Sean Goggins, Jonahz|
+|[Akshara P](#Akshara-P)|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Pratik Mishra](#Pratik-Mishra)|Machine Learning for Anomaly Detection in Open Source Communities|Elita Nelson, Sean Goggins, Gabe Heim|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
+
+
+## Mentees

```

Comment:
Thank you for merging all of the mentees. Could you please reverse the order, so that the latest projects are always at the top?